### ISSUE TITLE
[Feature Request] Variable "if" to check if MM is dealing with Java 1.8 or 9 GA in launching scripts

### ISSUE THREAD

**Issue by MGM175** [MegaMek/megamek#680]
Since recently Java 9 reached General Availability status, so it can be downloadable like the version 1.8.
As we know, Java 9 gives us LOTS of changes, including the obvious versioning change.

Is there a possibility to include something like:

```
If
java --version | grep "1.8" "9"
then "java -Xmx1024 -jar MegaMek.jar #if 1.8 value shows up
or
then "java --add-modules java.se.ee -Xmx1024 -jar MegaMek.jar #if value 9 shows up
```
because when I'll launch MM normally, Java 9 throws error at launch:

`Exception in thread "main" java.lang.NoClassDefFoundError: javax/xml/bind/JAXBException
	at megamek.MegaMek.<clinit>(MegaMek.java:71)
Caused by: java.lang.ClassNotFoundException: javax.xml.bind.JAXBException
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:582)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:185)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:496)
	... 1 more`

Tested on:
```
"java --version" output

openjdk 9.0.0.15
OpenJDK Runtime Environment (Zulu build 9.0.0.15+181)
OpenJDK 64-Bit Server VM (Zulu build 9.0.0.15+181, mixed mode
```

**Comment by pheonixstorm** [comment]
Need to know what version of MM you are using as well as a copy of the actual log file

**Comment by arlith** [comment]
That information isn't necessary.  He's asking that we update the start script.  Java 9 has broken out some of the standard library into modules that don't always load, so what he's indicating is that Java 9 doesn't automatically load a module that is required for MM.  So, we need to add a parameter to enable the java.se.ee module so it doesn't crash.  This is easy for Linux (it's mostly just making the changes he suggested in the shell script), but I'm less sure how for other OSes.  I guess for Windows, we can update what's in the batch "exe" file.

**Comment by arlith** [comment]
Added a fix in 238a82d.  I didn't test this with Java 9, but I think it should work.  The script still works for Java 8, and as long as the version check works like I think it does, it should work for Java 9.

**Comment by MGM175** [comment]
@arlith I did some testing on several versions of Java 9 (OpenJDK, Azul Zulu & Oracle Java) & recently on the very early build of the Java 10 (build 33 - http://jdk.java.net/10/). Updated script works flawlessly :+1: 

Proof below:
![zrzut ekranu z 2017-12-01 22-38-52](https://user-images.githubusercontent.com/20325180/33513917-ad3a125a-d74a-11e7-943d-b013077dc6d5.png)
